hack/render-manifests.go

```diff
@@ -22,7 +22,7 @@ func main() {
 	}
 
 	if len(os.Args) < 6 {
```

Comment:
You add another parameter [handler/operator] , don't you need to update if statement? os.Args < 7 ?